doc/rados/operations/erasure-code-profile.rst

```diff
@@ -96,7 +96,9 @@ Where:
 ``--force``
 
 :Description: Override an existing profile by the same name, and allow
-              setting a non-4K-aligned stripe_unit.
+              setting a non-4K-aligned stripe_unit. The former can be
+              dangerous, and thus ``--yes-i-really-mean-it`` must be
```

Comment:
s/must be/must be used/
or
s/must be/must be invoked/

Reply:
Thanks!

Reply:
Suggest s/The former/This/

Reply:
The intention was call _dangerous_ just the overriding of existing, not setting an exotic stripe size.

Reply:
I suggest /The former/Overriding an existing profile/